flexx/ui/widgets/_button.py

```diff
@@ -115,11 +121,18 @@ def _init_phosphor_and_node(self):
             self.phosphor = self._create_phosphor_widget('button')
             self.node = self.phosphor.node
             self.node.addEventListener('click', self.mouse_click, 0)
-        
+
         @event.connect('text')
         def __text_changed(self, *events):
             self.node.innerHTML = events[-1].new_value
 
+        @event.connect('disabled')
+        def __disabled_changed(self, *events):
+            if events[-1].new_value:
+                self.node.setAttribute("disabled", "disabled")
```

Comment:
Setting `"disabled"` to `"disabled"` wouldn't that be a bool or does the value not matter anyway?